### PR TITLE
fix #21941 call_trigger

### LINKED ISSUE #21941: workflow : public online sign does not call PROPAL_CLOSE_SIGNED trigger
### Bug

You have to enable workflow module AND activate "auto create order when propal is signed".

Then if you clic on dolibarr backend button "validate" propal it make an order -> perfect.

BUT if you send to your customer the online "sign" link -> the trigger is not run then workflow does not work.

### Environment Version

_No response_

### Environment OS

_No response_

### Environment Web server

_No response_

### Environment PHP

_No response_

### Environment Database

_No response_

### Environment URL(s)

_No response_

### Expected and actual behavior

_No response_

### Steps to reproduce the behavior

_No response_

### Attached files

_No response_

### PR DESCRIPTION
**Pull request by rycks**
# FIX #21941 : call trigger on public propal sign

### CODE REVIEW

File: htdocs/public/onlinesign/newonlinesign.php [hunk 1]
```diff
@@ -452,13 +452,34 @@
 			print '<br>';
 			if ($message == 'signed') {
 				print '<span class="ok">'.$langs->trans("PropalSigned").'</span>';
+				//fix #21941 run trigger
+				if (method_exists($object, 'call_trigger')) {
```
Comment (eldy) [inline htdocs/public/onlinesign/newonlinesign.php:456]: Why calling the method into the public page and not into the method that set the status so into 
core/ajax/onlineSign.php

Comment (rycks) [reply htdocs/public/onlinesign/newonlinesign.php:456]: why .... hmmm because i have to make a choice 🔀  but it seems to be the bad one i've made, sorry 👯

Comment (rycks) [reply htdocs/public/onlinesign/newonlinesign.php:456]: so ? would you prefer that i switch that code from newonlinesign.php to ajax/onlineSign.php ?

Comment (eldy) [reply htdocs/public/onlinesign/newonlinesign.php:456]: Yes trigger must be nearest the code that justify its execution.

Comment (rycks) [reply htdocs/public/onlinesign/newonlinesign.php:456]: yes sir ! i will do it now

Comment (rycks) [reply htdocs/public/onlinesign/newonlinesign.php:456]: ok, i've moved one trigger to ajax/onlineSign.php but for the other (refused) there is no other place than newonlinesign.php
... sorry :)

commit & PR update will become in some sec / min depends on my git-power !

Comment (rycks) [reply htdocs/public/onlinesign/newonlinesign.php]: PR is clean and ready :)

**Pull request merged**